Code/Framework/AzCore/AzCore/Debug/Profiler.h

```diff
@@ -318,6 +58,40 @@ namespace AZStd
 
 namespace AZ
 {
+    class ProfileScope
+    {
+    public:
+        static uint32_t GetSystemID(const char* system);
+
+        template<typename... T>
+        static void BeginRegion([[maybe_unused]] const char* system, [[maybe_unused]] char const* eventName, [[maybe_unused]] T const&... args)
+        {
+            // TODO: Verification that the supplied system name corresponds to a known budget
+#if defined(USE_PIX)
+            PIXBeginEvent(PIX_COLOR_INDEX(GetSystemID(system) & 0xff), eventName, args...);
```

Comment:
So we added these markers within DX12 RHI backend as well for commandlists as well as pass markers. I wonder if they will conflict with this. Look at Scope::Begin and CommandList::Open for that code within dx12 RHI

Reply:
No conflict from what I can tell.